src/main/java/com/sekwah/advancedportals/bukkit/portals/Portal.java

```diff
@@ -202,6 +201,8 @@ public static String create(Location pos1, Location pos2, String name, String de
 
         portalData.getConfig().set(name + ".destination", destination);
 
+        portalData.getConfig().set(name + ".cooldowndelay", cooldown);
```

Comment:
This is not needed if you are using the provided extraArgs implmentation such as the command tags.